test/e2e/task/start_test.go

```diff
@@ -83,12 +83,30 @@ Waiting for logs to be available...
 		assert.Assert(t, is.Regexp(expected, res.Stdout()))
 	})
 
+	t.Run("Start TaskRun using tkn task start command with --use-param-defaults", func(t *testing.T) {
+		res := tkn.Run("task", "start", "read-task",
+			"-i=source="+tePipelineGitResourceName,
+			"--use-param-defaults",
+			"--showlog",
+			"true")
+
+		res.Assert(t, icmd.Expected{
+			ExitCode: 0,
+			Err:      icmd.None,
+		})
+		assert.Assert(t, is.Regexp("TaskRun started:.*", res.Stdout()))
+	})
+
 	t.Run("Get list of TaskRuns from namespace  "+namespace, func(t *testing.T) {
 		res := tkn.Run("taskrun", "list")
 		expected := e2e.ListAllTaskRunsOutput(t, c, false, map[int]interface{}{
 			0: &e2e.TaskRunData{
 				Name:   "read-task-run-",
-				Status: "Succeeded",
+				Status: "",
+			},
+			1: &e2e.TaskRunData{
+				Name:   "read-task-run-",
+				Status: "",
```

Comment:
What is the reason for removing Status?

Reply:
There is a function used in e2e testing for pipelines that waits until the TaskRun finishes:

https://github.com/tektoncd/pipeline/blob/2e737622b576c373894aa4ac2d6f83dc46096cb3/test/v1alpha1/wait.go#L72

Example of using it:

https://github.com/tektoncd/pipeline/blob/89077a45f003f47287a5ea948cda66f506bf5932/test/taskrun_test.go#L75

Reply:
Thanks @danielhelfand for the suggestion. :)
Added the same.